src/auto-save-service.ts

```diff
@@ -83,3 +83,21 @@ export async function deleteAutoSaveData(
   const key = getAutoSaveKey(formId, autoSaveKey)
   return utilsService.removeLocalForageItem(key)
 }
+
+/**
+ * Delete all auto saved data.
+ *
+ * #### Example
+ *
+ * ```js
+ * await autoSaveService.deleteAllAutosaveData()
+ * ```
+ *
+ * @returns
+ */
+export async function deleteAllAutosaveData() {
+  const keys = await utilsService.getLocalForageKeys('AUTO_SAVE_')
```

Comment:
Can we please make `'AUTO_SAVE_'` a variable and use it in the `getAutoSaveKey()` function as well to avoid this breaking if one changes without the other changing.